src/test/java/org/prebid/server/bidder/BidderCatalogTest.java

```diff
@@ -34,7 +36,7 @@
     @Test
     public void isValidNameShouldReturnTrueForKnownBidder() {
         // given
-        bidderDeps = BidderDeps.of(BIDDER, null, null, null, null, null);
+        bidderDeps = BidderDeps.of(BIDDER, Collections.emptyMap(), null, null, null, null, null);
```

Comment:
Unlike production code we do allow and encourage reasonable static imports in tests as it improves readability.